index.html

```diff
@@ -42,13 +44,218 @@ <h1>Organic</h1>
         </header>
     </div>
 
-
     
     <!---------How it works section----------->
 
-    
+  <section>
+       <div class="goods">
+            <h2>How it works?</h2>
+            <p>The principle of how markup works is as follows: 
+               You enter the address of a website or page into the address bar. 
+               The browser sends a request to this address and receives a file in HTML format. 
+               The code from the resulting file is sequentially converted into visual 
+               objects.</p>
```

Comment:
It is better to keep indents on every level the same. 'section' indent is 2 spaces, but 'div' and 'h2', 'p' are 4 - it is not good.
![image](https://github.com/Tomikander/organic_landing/assets/65491655/eea1cad1-fea2-4d38-a40b-2789b793651c)
And multiline text should start on a new line, and the closing tag should be on a new line also (with the same indent as the opening tag)